pages/lazer/getting-started.mdx

```diff
@@ -1,3 +1,155 @@
+import { Callout, Steps } from "nextra/components";
+
 # Getting Started with Pyth Lazer
 
-Please refer to the how-to guides to get started.
+Pyth Lazer is a high-performance, low-latency service that provides real-time financial market data.
+This guide will walk you through setting up and running a basic JavaScript example to subscribe to Lazer price feeds.
+
+## Prerequisites
+
+Before getting started, make sure you have the following installed:
+
+- **Node.js** (version 18 or higher)
+- **pnpm** package manager
+- **Git** for cloning the examples repository
+- A **Lazer Access Token** - see [How to Acquire an Access Token](./acquire-access-token) if you don't have one
+
+<Steps>
+
+### Clone the Examples Repository
+
+First, clone the Pyth examples repository which contains the JavaScript SDK example:
+
+```bash copy
+git clone https://github.com/pyth-network/pyth-examples.git
+cd pyth-examples/lazer/js
+```
+
+### Install Dependencies
+
+Install the required dependencies using pnpm:
+
+```bash copy
+pnpm install
+```
+
+This will install `@pythnetwork/pyth-lazer-sdk`, which is the main Lazer SDK.
+
+### Configure Your Access Token
+
+Set your Lazer access token as an environment variable:
+
+```bash copy
+export ACCESS_TOKEN=your_actual_token_here
```

Comment:
Can we call it API key? or just say the user has to put the api key. Access token === API_KEY

Reply:
it's called an access token everywhere else